app/schema.json

```diff
@@ -4,31 +4,105 @@
   "title": "HiGlass viewconf",
   "type": "object",
   "additionalProperties": false,
+  "required": [],
   "properties": {
     "editable": { "type": "boolean" },
     "zoomFixed": { "type": "boolean" },
     "exportViewUrl": { "type": "string" },
     "trackSourceServers": {
       "type": "array",
-      "items": { "type": "string" }
+      "items": { "type": "string" },
+      "minLength": 1
```

Comment:
You don't necessarily need to have a track source server. You could have it empty if you don't want to allow people to add or remove tracks.

Reply:
But then again, I suppose if it's going to be empty, you might as well not include the section at all.